docs/release_notes/ibexa_dxp_v4.4.md

```diff
@@ -0,0 +1,146 @@
+---
+description: Ibexa DXP v4.4 adds the improvements to the Welcome Page, All-new Ibexa Commerce packages and Fastly.
+---
+
+# Ibexa DXP v4.4
+
+**Version number**: v4.4
+
+**Release date**: January 31, 2023
+
+**Release type**: [Fast Track](https://support.ibexa.co/Public/service-life)
+
+## Notable changes
+
+### New welcome page
+
+You will be welcomed to the Ibexa Digital Experience Platform by a newly designed welcome page.
+
+![New Welcome Page](4.4_welcome_page.png)
+
+### All-new Ibexa Commerce packages [[% include 'snippets/commerce_badge.md' %]]
+
+This release deprecates all Commerce packages that you've known from previous releases 
+and brings a redesigned and reconstructed Commerce offering.
+
+As part of this effort, two all-new components have been created: Cart and Checkout, 
+that you can use to build your own e-commerce presence. 
+
+![The new cart view](img/4.4_new_cart.png "The new cart view")
+
+Another addition is the Storefront component that provides a set of basic components 
```

Comment:
@dabrt we repeat here the word "component", maybe we can just say "Another addition is the Storefront that provides a set of basic components.."